packages/backend/package.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "@sourcebot/backend",
+    "version": "0.1.0",
+    "private": true,
+    "main": "index.js",
+    "type": "module",
+    "scripts": {
+        "dev:watch": "yarn generate:types && tsc-watch --onSuccess \"yarn dev --configPath ../../config.json --cacheDir ../../.sourcebot\"",
```

Comment:
We probably want to double check that these arguments for `--configPath` and `--cacheDir` work when calling `dev:watch` from the workspace root.